modules/rosa-cluster-autoscaler-ui-settings.adoc

```diff
@@ -16,25 +18,45 @@ The tables explain all the configurable UI settings when using cluster autoscali
 |Type or Range
 |Default
 
-|`balanceSimilarNodeGroups`
+|`balance-similar-node-groups`
 |If `true`, this setting automatically identifies node groups with the same instance type and the same set of labels and tries to keep the respective sizes of those node groups balanced.
 |`boolean`
 |false
 
-|`logVerbosity`
-|Sets the autoscaler log level. Default value is 1, level 4 is recommended for debugging and level 6 enables almost everything.
-|`integer`
-|1
+|`skip-nodes-with-local-storage`
+|If `true`, the cluster autoscaler never deletes nodes with pods with local storage, e.g. EmptyDir or HostPath.
+|`boolean`
+|true
 
-|`maxPodGracePeriod`
+|`max-pod-grace-period (max-graceful-termination-sec)`
```

Comment:
This parameter has a different name in the API and in Kubernetes, that's why in the UI, we added the two labels with the format `<api-label> (<k8s-label>)`

@osherdp WDYT? Depending on which doc the user is familiar with, they would know one of the names.

Reply:
Our direct API here is openshift, as it's abstracting away some of the fields Kubernetes autoscaler is using
So I think we should only use the names of OCM/OpenShift API

Reply:
Makes sense. I'll create a small issue to remove the k8s name from the UI.

Reply:
So which name is staying? `max-pod-`grace-period`?

Reply:
Yes